### DISCUSSION TITLE
Submitting does nothing for Student verification

### DISCUSSION THREAD

**Discussion by marc726** [community/community#69483] [GitHub Education]
### Hi everyone,

When I submit my verification for enrollment I do not get a confirmation. After it's submitted, I'm asked to immediately submit proof of my academic status again. Is there something I'm missing here? I've see the confirmation screen from other people before.

**Comment by ai-xiaihai** [comment]
> ### Hi everyone,
> When I submit my verification for enrollment I do not get a confirmation. After it's submitted, I'm asked to immediately submit proof of my academic status again. Is there something I'm missing here? I've see the confirmation screen from other people before.

I am having the same issue

**Reply by wesleydecezere** [reply]
Eu também :<

**Comment by santoshvandari** [comment]
If you are experiencing an issue where submitting your verification for student enrollment does not result in a confirmation and you are prompted to submit proof again, there could be a few potential reasons for this issue. Here are some steps you can take to troubleshoot:

* **Clear Browser Cache:** Start by clearing your browser's cache and cookies. Sometimes, cached data can interfere with form submissions. After clearing the cache, try the verification process again.
* **Use a Different Browser:** Try using a different web browser to submit your verification. Sometimes, browser-specific issues can affect the behavior of web forms.
* **Check for Errors:** Ensure that you have filled out all required fields accurately in the verification form. Any errors or missing information might prevent the form from being submitted successfully.
* **Internet Connection:** Ensure that you have a stable internet connection while submitting the verification. A poor or unstable connection can cause issues with form submissions.
* **Disable Browser Extensions:** Browser extensions or add-ons can sometimes interfere with web forms. Try disabling any extensions temporarily and attempt the verification process again.
* **Try a Different Device:** If possible, try the verification process on a different device (e.g., another computer or a mobile device) to see if the issue persists.
* **Contact GitHub Support:** If the issue continues to persist after trying these steps, consider reaching out to GitHub Support or the GitHub Education support team for assistance. They can investigate the issue further and provide guidance or solutions specific to your situation.

**It's important to ensure that all the information you provide during the verification process is accurate and complete. If the problem persists, GitHub's support teams should be able to assist you in resolving the issue and confirming your student status.**

**Comment by RiddheshMore** [comment]
Even i am getting the same error !!!

**Comment by eliz-amst** [comment]
You should see your application pending on a side of the webpage, you can submit another request if you wish, as long as its pending you just wait fro GitHub to approve or reject your application
